src/disp/program.c

```diff
@@ -2677,302 +2677,331 @@ MVMuint32 MVM_disp_program_record_end(MVMThreadContext *tc, MVMCallStackDispatch
             MVM_oops(tc, "Unimplemented dispatch program outcome kind");
     }
 }
+#undef MVM_CGOTO
+#define MVM_CGOTO 0
+#if MVM_CGOTO
+#define DISPATCH(op)
+#define OP(name) name
+#define NEXT do { \
+    if(i < dp->num_ops) {       \
+        op = dp->ops[i++];      \
+        goto *LABELS[op.code];  \
+    } else                      \
+        goto acception;         \
+    } while(0)
```

Comment:
I think we can get rid of the `i < dp->num_ops` check here and just put a `return 1` into the terminals; every dispatch program ends with one of the `MVMDispOpcodeResult*` ops.

Reply:
`goto acception` can be replaced with `return 1`,  I don't understand the part of getting rid of `i < dp->num_ops` because I know nothing about dispatcher :)